### DISCUSSION TITLE
¿De cuánto es la sangría del output?

### DISCUSSION THREAD

**Discussion by Utmite** [IIC2133-PUC/2024-2#32] [Tarea 0] [answered]
Hola, buenas noches.

¿La sangría hacia la derecha debe ser de 3 o 4 espacios?

Gracias por su atención.

**Comment by waknaudt** [comment]
Creo que son tabs, pero lo que entiendo diff - b no considera los espacios asi que parece que da igual.

**Comment by tomastrivino** [comment, marked as answer]
Hola,
efectivamente es lo que mencionan en el otro comentario. Puedes utilizar `printf("\tEspacio\n");` para imprimir algo de la forma
```c
    Espacio
```
Éxito en la tarea.

**Reply by Utmite** [reply]
Muchas gracias el "\t" me arreglo todos mis problemas :3
